src/devtools/profiler.js

```diff
@@ -0,0 +1,36 @@
+let profileEntries = [];
+
+export function getProfilerData() {
+  return profileEntries;
+}
+
+/**
+ *
+ * @type {'application' | 'parcel' | 'routing'} ProfileType
+ *
+ * @param {ProfileType} type
+ * @param {String} name
+ * @param {number} start
+ * @param {number} end
+ */
+export function addProfileEntry(
+  type,
+  name,
+  kind,
+  start,
+  end,
+  operationSucceeded
+) {
```

Comment:
It's interesting that you take these in as separate params (which require remembering the order), but in the end store them as object properties in the end. Would it be easier to just make this an object from the get-go?

Reply:
I did them as separate params with the idea being it would keep bundle size down. It's true that i later put them in an object, but my thought was that at least the property names are only in one place rather than in every place that adds a profile entry?